react-common/styles/share/share.less

```diff
@@ -312,6 +313,11 @@
     .gif-recorder-content .thumbnail-controls {
         padding: 0 2rem;
     }
+
+    .embed.mobile-portrait-hidden {
+        color: #323130 !important;
```

Comment:
why the use of `!important` here? can it be avoided by making the selector more specific?

important makes maintaining css harder (though i know we use it a little too often in our codebase already 😆)

Reply:
It looks like I can not have the important for the background color, but the text color needs to be important because of semantic ui stuff: 
![image](https://github.com/microsoft/pxt/assets/49178322/a2fcd5ce-6ddd-4165-8500-5d4dae011ba4)